.gitignore

```diff
@@ -1,2 +1,2 @@
 /node_modules
-config.json
\ No newline at end of file
+lib/config.json
```

Comment:
stuff in a lib folder is usually static, i think you have this here cause of a line in the watson.js file, let's get that looking at the top level where config related files usually live